api/rs/slint/mcu.md

```diff
@@ -233,28 +229,33 @@ loop {
 ### The Renderer
 
 In desktop and embedded environments, Slint typically uses operating system provided APIs to render the user interface using the GPU.
-In contrast, most MCUs don't have GPUs. Instead, the all the rendering is done by software on the CPU. This is called software rendering, and Slint provides a SoftwareRenderer for this task.
+In contrast, most MCUs don't have GPUs. Instead, software rendering is used where all rendering is done by software on the CPU.
+Slint provides a SoftwareRenderer for this task.
 
-In the previous example, we've instantiated a [`slint::platform::software_renderer::MinimalSoftwareWindow`]. This struct implements the
-`slint::platform::WindowAdapter` trait and also holds an instance of a [`slint::platform::software_renderer::SoftwareRenderer`]. You obtain access to it
+In the earlier example, we've instantiated a [`slint::platform::software_renderer::MinimalSoftwareWindow`]. This struct implements the
+`slint::platform::WindowAdapter` trait and also holds an instance of a [`slint::platform::software_renderer::SoftwareRenderer`]. You access it
 through the callback parameter of the [`draw_if_needed()`](MinimalSoftwareWindow::draw_if_needed) function.
 
-Depending on the amount of RAM your MCU is equipped with, and the kind of screen that is attached, you can choose between two different ways of using the renderer:
+Depending on the amount of RAM your MCU has, and the kind of screen attached, you can choose between two different ways of using the renderer:
 
- * Use the [`SoftwareRenderer::render()`] function if you have enough RAM to allocate one, or even two, copies of the entire screen (also known as
+ * Use the [`SoftwareRenderer::render()`] function if you have enough RAM to hold one, or even two, copies of the entire screen (also known as
```

Comment:
Why did you replace "allocate" with "hold"?

I think "allocation" in the context of "RAM" is idiomatic, while "holding" isn't.